src/clj/rems/workflow/dynamic.clj

```diff
@@ -892,7 +1022,10 @@
       (is (= #{}
              (possible-commands "assistant" draft)))
       (is (= #{}
-             (possible-commands "somebody else" draft))))
+             (possible-commands "somebody else" draft)))
+      (testing "when there are invited members"
+        (is (= #{::submit ::invite-member ::uninvite-member}
+               (possible-commands "applicant" (assoc draft :invited-members [{:name "Some One" :email "some.one@example.org"}]))))))
```

Comment:
Please refactor this so that the application is produced from events with `apply-events`, similar to all the other tests; don't assoc stuff directly to `draft`.

Reply:
Ok I based my tests on earlier code that didn't use the events.